live-examples/js-examples/intl/meta.json

```diff
@@ -60,6 +60,12 @@
             "title": "JavaScript Demo: Intl.ListFormat",
             "type": "js"
         },
+        "intlLocalePrototypeMaxmimize": {
+            "exampleCode": "./live-examples/js-examples/intl/intl-locale-prototype-maximize.html",
+            "fileName": "intl-locale-prototype-maximize.html",
+            "title": "JavaScript Demo: Intl.Locale.prototype.maximize",
```

Comment:
This should be "JavaScript Demo: Intl.Locale.prototype.maximize()".

Reply:
Changes made!